src/m365/spo/commands/user/user-get.ts

```diff
@@ -98,6 +156,21 @@ class SpoUserGetCommand extends SpoCommand {
     else if (args.options.loginName) {
       requestUrl = `${args.options.webUrl}/_api/web/siteusers/GetByLoginName('${formatting.encodeQueryParameter(args.options.loginName)}')`;
```

Comment:
This was not really a part of your issue, but currently we always set the entire url. Let's start with requestUrl being args.options.webUrl/_api/web and then add the rest